build_tools/cmake/build_and_test_asan.sh

```diff
@@ -22,7 +22,14 @@ BUILD_DIR="${1:-${IREE_ASAN_BUILD_DIR:-build-asan}}"
 IREE_ENABLE_ASSERTIONS="${IREE_ENABLE_ASSERTIONS:-ON}"
 
 source build_tools/cmake/setup_build.sh
-source build_tools/cmake/setup_ccache.sh
+if command -v ccache &> /dev/null
+then
+  source build_tools/cmake/setup_ccache.sh
+fi
+if command -v sccache &> /dev/null
+then
+  source build_tools/cmake/setup_sccache.sh
+fi
```

Comment:
Interesting... I like the idea of supporting either ccache or sccache. Is this checking for the existence of either tool? We would want to check CLI clags or environment variables instead, since both tools can be installed at the same time.